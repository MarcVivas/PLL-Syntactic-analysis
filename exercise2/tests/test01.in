// Test arithmetic expressions
3 + 4 * 2 - 1;
(3 + 4) * 2 - 1;
10 / 2 * 3;
2 * 10 / 4;
-5 + 8;
10 % 3;
