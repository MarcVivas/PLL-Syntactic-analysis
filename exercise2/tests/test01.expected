10
13
15
5
3
1
